includes/class.llms.template.loader.php

```diff
@@ -313,6 +315,12 @@ public function restricted_by_membership( $info ) {
 			$msg      = '';
 			$redirect = '';
 
+			if ( 'yes' === $membership->get( 'restriction_add_notice' ) ) {
```

Comment:
When evaluating yes/no values use `llms_parse_bool()`. Not a necessary change to make but using this method will allow greater consistency across the code base where yes/no values are stored. 

https://developer.lifterlms.com/reference/functions/llms_parse_bool/

Eg: `if ( llms_parse_bool( $membership->get( 'restriction_add_notice' ) ) ) {`

Reply:
Yes, you're right, for some reason I keep on forgetting about that util.

Reply:
No worries, just alerting you!